package.json

```diff
@@ -92,10 +105,12 @@
     "http-proxy-middleware": "2.0.6",
     "jest": "28.1.1",
     "mocked-env": "1.3.5",
+    "optional-js": "2.3.0",
```

Comment:
> unmaintained
Or: Because this lib ist just feature complete and doesn't have any dependencies ist is... stable! :scream:

Reply:
looking at the maintainers github profile, I'm not sure how maintained it can be: Literally no activity in 2021 and 2022. In 2020 there was one busy Wednesday.

His website seems also gone. So… feature complete but unmaintained?